src/screens/DiscoveryScreenV2/elements/DiscoveryTab.js

```diff
@@ -44,7 +44,7 @@ const DiscoveryTab = ({onChangeScreen, selectedScreen = 0, tabs}) => {
               android_ripple={{color: COLORS.balance_gray}}
               style={[
                 styles.tabItem(route.name === 'Followings' ? 3 : 4),
-                index === selectedScreen ? styles.underlineFocus : {}
+                index === selectedScreen ? styles.underlineActive : styles.underlineInactive
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the style conditionally applied based on `index === selectedScreen` is a good one. It improves readability by clearly defining what styles are applied when the condition is true or false. However, ensure that the styles `underlineActive` and `underlineInactive` are defined correctly and provide the desired visual effect.
```diff
-                index === selectedScreen ? styles.underlineFocus : {}
+                index === selectedScreen ? styles.underlineActive : styles.underlineInactive
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->